README.rst

```diff
@@ -43,6 +43,53 @@ To install from github::
 
     $ pip install git+git://github.com/robinedwards/neomodel.git@HEAD#egg=neomodel-dev
 
+Authentication
+--------------
+Please note that if you are utilizing Neo4j version 2.2 or newer there are
+some additional setup steps necessary relating to authentication. As of version 2.2
+Neo4j authentication is activated by default on new instances. Please follow the
+outstanding documentation provided by py2neo's Authentication_
+section to setup new credentials. If you are utilizing a hosted service this
+is most likely already taken care for you.
+
+.. _Authentication: http://py2neo.org/2.0/essentials.html#authentication
+
+Upgrading 1.x to 2.x
+====================
+There is one modification that is necessary when transitioning from version
+1.x to 2.x relating to direct cypher queries. In version 1.x performing a
+cypher query always returned at least one array within an array. This enabled
+some assumptions to be made when querying for single object directly.
+Since you could assume the result would always be a list within a list
+the result of a response could be safely accessed like the following::
+
+    query = 'MATCH (a:Profile {username: "%s"}) RETURN a' % (username)
+    res, col = db.cypher_query(query)
+    end_result = res[0][0]
+
+If there wasn't a Profile with the given username nothing would fail out and
+``end_result`` would be set to an empty list.
+
+In version 2.x we lean a bit more on py2neo to handle the response. This results
+in what py2neo refers to as a RecordList_.
+A RecordList is a list of Record_ objects which provide
+a few more capabilities on accessing the results. The draw back is
+that now accessing the above query at ``res[0][0]`` will result in an ``IndexError``
+due to the new response being an empty ``RecordList``. Py2neo has however provided
+a solution to this which is the one_ method. This method can be called on the
+cypher query response to achieve the old ``res[0][0]`` functionality. So in
+version 2 the above query would look like::
+
+    query = 'MATCH (a:Profile {username: "%s"}) RETURN a' % (username)
+    res, col = db.cypher_query(query)
+    end_result = res.one()
```

Comment:
Not a note on the docs but more the implementation. Do we want to expose these py2neo objects directly from the neomodel API or do we want to do decouple them? I've always considered different backends or even embedding our own client inside neomodel eventually.